### ISSUE TITLE
Make sure traditional login still works.

### ISSUE THREAD

**Issue by lancejpollard** [lancejpollard/authlogic-connect#24]

**Comment by jeroenhouben** [comment]
@viatropos
Am I right in thinking that right now you can ONLY register / signin using external providers and not a locally stored username/password?

I have this use case where default Authlogic behaviour (no add-ons) is in place, but I'd like to make it so that users can, if they want, register/login with Facebook, Google etc etc. I think the OpenID add-on works that way.

**Comment by badnaam** [comment]
I would like to know this too, will regular site username/password login still work?

**Comment by jspooner** [comment]
Yeah I'm trying to solve this same problem.  I have the basic authlogic user/password right now and I'd like to add this authlogic-connect gem. 

The problem I'm having is that a new user record is created when an existing user logins in with facebook.  Since oauth just returns with a token and key I'm unsure how to prevent this second record from being created.  

I guess that after the second user is created we can discover the users email address from facebook then merge records.  Does that sound reasonable?
